examples/Tilt-Compensated Compass Example

```diff
@@ -0,0 +1,77 @@
+## Tilt Compensated Compass using Accelerometer and Magnetometer
```

Comment:
the header always have the following information

```python
# SPDX-FileCopyrightText: 2021 <Your Name>
# SPDX-License-Identifier: MIT
```

If you do not want to use your name you could use your Github User